src/mktplinkfw2.c

```diff
@@ -149,6 +149,12 @@ static struct flash_layout layouts[] = {
 		.kernel_la	= 0x80000000,
 		.kernel_ep	= 0x80000000,
 		.rootfs_ofs	= 0x140000,
+	}, {
+		.id		= "16Mmtk",
+		.fw_max_len	= 0xf90000,
```

Comment:
```suggestion
		.fw_max_len	= 0xfa0000,
```

Reply:
Aha, yes I miscalculated by 64kB.

Reply:
@DragonBluep mhhh the other 16m entry are f9... why this should be fa?

Reply:
This is `firmware` partition size. See @zamaudio PR https://github.com/openwrt/openwrt/pull/15610/files. ath79 target has different partition layout.